app/views/kaui/audit_logs/_show_history_modal.html.erb

```diff
@@ -0,0 +1,192 @@
+<div class="modal fade" id="showHistoryModal" tabindex="-1" role="dialog">
+  <div class="modal-dialog modal-lg" role="document">
+    <div class="modal-content column-block">
+      <div>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h1 class="modal-title">History</h1>
+      </div>
+      <div class="modal-body">
+        <input type="hidden" id="objectId">
+        <input type="hidden" id="objectType">
+        <input type="hidden" id="changeDate">
+        <div class="alert alert-info" role="alert">Loading data from server</div>
+        <div id="templatesPlaceHolder" style="display: flex;">
+          <div id="templateFieldsPlaceHolder"></div>
+          <div id="templatePlaceHolder"></div>
+        </div>
+      </div>
+    </div><!-- /.modal-content -->
+  </div><!-- /.modal-dialog -->
+</div><!-- /.modal -->
+
+<script id="history_template" type="text/template" >
+  {{#auditLogs}}
+  <div class="panel panel-{{selected}}" id="{{panelId}}">
+    <div class="panel-heading">{{changeType}} - {{changeDate}}</div>
```

Comment:
The first one (top left) is a bit weird (displaying ` - ` because `changeType` and `changeDate` are `nil`). Could we switch to a non-printable character instead just for that first column, e.g. `&nbsp;`?